polyfill/lib/calendar.mjs

```diff
@@ -875,8 +877,8 @@ const nonIsoHelperBase = {
   addCalendar(calendarDate, { years = 0, months = 0, weeks = 0, days = 0 }, overflow, cache) {
     const { year, month, day } = calendarDate;
     const addedMonths = this.addMonthsCalendar({ year: year + years, month, day }, months, overflow, cache);
-    days += weeks * 7;
-    const addedDays = this.addDaysCalendar(addedMonths, days, cache);
+    const initialDays = days + weeks * 7;
+    const addedDays = this.addDaysCalendar(addedMonths, initialDays, cache);
```

Comment:
This refactor was also needed in TS, but I left it in because I thought it made the code easier to follow. LMK if you'd like me to revert.

Reply:
removed this change; not important enough to port.